### ISSUE TITLE
Survey123 Connect ver 1.5.35 freezing (Not Responding)

### ISSUE THREAD

**Issue by bcgisjosh** [Esri/Survey123Community#509]
I downloaded the latest version of Survey123 Connect. The first survey I made was a success. 
But the next one I tried, the application freezes, becomes unresponsive. This happens when I try choosing an already created survey or when trying to create a new survey. It just freezes and even after five minutes, remains unresponsive. 

I went back to ver 1.3.something and it seems to be OK. 
Anyone else experiencing this?

**Comment by kevintheflesh** [comment]
I am having the same issue. I can't open my previously created survey or create new survey. I've tried to uninstall and reinstall 1.5.35 with no luck.

**Comment by survey123** [comment]
Could you please send your surveys. We would like to look into them.
Please send the entire survey folder (zipped) to us at survey123@esri.com

**Comment by bcgisjosh** [comment]
sent to you on Tuesday around noon (Pacific time)
Hopefully you can help.

**Comment by survey123** [comment]
Thanks to all for reporting this. 
Moving this issue into our next milestone (September 2016).
In the meantime, to keep things moving, go into the My Survey Designs folder and clean up all unwanted Surveys you may have. Delete the entire folders. Then use Survey123 Connect again. 
Turns out this issue is not caused by a particular survey, but by a resizing problem in Survey123. By changing the number of Surveys in the gallery, the problem goes away (only temporarily if you are not  lucky).  When it crashes, go into the My Survey Design folder and remove junk. If you have no junk, copy and paste a couple of folders (create junk).  Yeah I know... not pretty. We are working on this and try to resolve as soon as we can.

**Comment by bcgisjosh** [comment]
this didn't seem to fix it for me. I deleted two folders in my "My Survey Design" folder. But I still am unable to open up already completed surveys. The application still freezes. 

It also doesn't allow me to create new surveys. It will popup excel but the Survey123Connect freezes at "Converting XLS Form"

**Comment by kevintheflesh** [comment]
After much trial and error, I finally was able to publish my survey! I deleted all the surveys in the My Survey Designs folder and I deleted all my survey123 folders from AGOL without much luck. But once I deleted all these survey folders PLUS changed the form_title, form_id, and the instance_name in my survey settings tab, it worked. I think that part of my issue was the form_title was still referencing an old survey I built?
